packages/services/src/services/Translation/__tests__/Translation.test.js

```diff
@@ -76,7 +105,76 @@ describe('TranslationAPI', () => {
         origin: 'https://ibm.com',
       },
     });
-
     expect(response).toEqual(responseSuccess);
   });
+
+  it('should return a json with a recent timestamp', async () => {
+    const mockDate = new Date('2019');
+    const _Date = Date;
+    global.Date = jest.fn(() => mockDate);
+    global.Date.now = _Date.now;
```

Comment:
Going away from `new Date()` for generating timestamp seems to allow us to test with less mock like below:

```suggestion
    const mockDate = 1546300800000; // Epoch time of January 1, 2019 midnight UTC
    global.Date.now = jest.fn(() => mockDate);
```